src/build-template.js

```diff
@@ -17,16 +16,37 @@ module.exports.renderTemplate = function (changelogTemplate, data, version) {
   return injectMetadata(compiledTemplate(data), version);
 };
 
-module.exports.saveChangelogToFile = function (filePath, renderedTemplate) {
-  const fileDescriptor = fs.openSync(filePath, "a+");
+module.exports.saveChangelogToFile = function (
+  filePath,
+  renderedTemplate,
+  startString
+) {
+  let oldData = null;
+  if (startString) {
+    const [oldDataBeforeStartString, oldDataAfterStartString] = fs
```

Comment:
Maybe a little bit pedantic, but if `startString` occurs multiple times in the current file, you will loose the text after the second occurrence.
You could handle this case as follows:
```
const [oldDataBeforeStartString, ...oldDataAfterStartString] = fs
  .readFileSync(filePath)
  .toString()
  .split(startString);
oldData = oldDataAfterStartString.join(startString)
```

Reply:
@buchi 👍 updated